### PR TITLE
homework/lesson-10: add the answer of js question

### PR DESCRIPTION
**Pull request by nguyenvanbapc66**

### CODE REVIEW

File: homework/ba-nguyen/lesson-10/js-5.md [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+
+```html
+<tab-panel>
+  <div data-tabname="one">Tab one</div>
+  <div data-tabname="two">Tab two</div>
+  <div data-tabname="three">Tab three</div>
+</tab-panel>
+<script>
+  // Keeps track of the currently selected tab.
+  let activeTab;
```
Comment (nguyenvanbapc66) [inline homework/ba-nguyen/lesson-10/js-5.md:91]: I don't have the answer so i copy from your solution

File: homework/ba-nguyen/lesson-10/js-5.md [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+
+**Question 7: What's the output?**
+
+```js
+const handler = {
+  set: () => console.log("Added a new property!"),
+  get: () => console.log("Accessed a property!"),
+};
+
+const person = new Proxy({}, handler);
```
Comment (nguyenvanbapc66) [inline homework/ba-nguyen/lesson-10/js-5.md:206]: Thaycacac: can you tell me this Proxy in reality applied for?

File: homework/ba-nguyen/lesson-10/js-5.md [hunk 3]
```diff
@@ -0,0 +1,274 @@
...
+- D: Object.assign(person, { age: 21 })
+
+> Explain: The method **seal** of the Object is mean that you can't modify the existing properties and you can't add new properties
+
+**Question 9: What's the output?**
+
+```js
+const add = (x) => x + x;
+
+function myFunc(num = 2, value = add(num)) {
```
Comment (nguyenvanbapc66) [inline homework/ba-nguyen/lesson-10/js-5.md:237]: thaycacac: new thing to me, can you give the keyword so i can continue researching?

File: homework/ba-nguyen/lesson-10/js-5.md (hunk 1 above)
Comment (thaycacac) [reply homework/ba-nguyen/lesson-10/js-5.md:91]: no problem

File: homework/ba-nguyen/lesson-10/js-5.md (hunk 2 above)
Comment (thaycacac) [reply homework/ba-nguyen/lesson-10/js-5.md:206]: some example for you [https://blog.logrocket.com/practical-use-cases-for-javascript-es6-proxies/](https://blog.logrocket.com/practical-use-cases-for-javascript-es6-proxies/)

File: homework/ba-nguyen/lesson-10/js-5.md (hunk 3 above)
Comment (thaycacac) [reply homework/ba-nguyen/lesson-10/js-5.md:237]: just javascript runtime

File: homework/ba-nguyen/lesson-10/js-5.md [hunk 4]
```diff
@@ -0,0 +1,274 @@
...
+  get: () => console.log("Accessed a property!"),
+};
+
+const person = new Proxy({}, handler);
+
+person.name = "Lydia";
+person.name;
+```
+
+- `A: Added a new property!` => **Answer**
```
Comment (thaycacac) [inline homework/ba-nguyen/lesson-10/js-5.md]: wrong answer

**Review by thaycacac — APPROVED** [review]

**Comment by thaycacac** [comment]
@quanghoang33 FE part LGTM

**Pull request merged**